core/app/models/spree/order/checkout.rb

```diff
@@ -268,7 +267,7 @@ def attach_country_id(attributes)
           end
 
           def country_id_by_iso(iso)
-            Spree::Country.find_by(iso: iso&.upcase)&.id
+            Spree::Country.find_by(iso: iso)&.id || Spree::Country.find_by(iso3: iso)&.id
```

Comment:
@przemosk we can ommit the `Spree` namespace here

Reply:
Also, we can use one SQL query for this with the `.or` method

https://blog.bigbinary.com/2016/05/30/rails-5-adds-or-support-in-active-record.html